### PR TITLE
Update gear info and images

### PR DESCRIPTION
**Pull request by RvanB**

### CODE REVIEW

File: client/components/public/Gear.jsx [hunk 1]
```diff
@@ -246,40 +244,20 @@ Gear = class Gear extends Component {
...
-                        </List.List>
-                      </List.Item>
+                      <List.Item>For any shirt (T-shirt or sweatshirt): $25/shirt</List.Item>
                     </List.List>
                   </List.Item>
                 </List>
               </Grid.Column>
               <Grid.Column width={5}>
-                  <Image src='https://gph-distributed.s3.us-west-2.amazonaws.com/2022/gear/shirt_design.png'/>
+                  <Image src='https://gph-distributed.s3.us-west-2.amazonaws.com/2023/gear/shirt_design.png'/>
```
Comment (richardgolding) [inline client/components/public/Gear.jsx]: Should this reference the s3_prefix const defined above?

File: client/components/public/Gear.jsx [hunk 2]
```diff
@@ -229,7 +227,7 @@ Gear = class Gear extends Component {
           <PuzzlePageTitle title="Gear" />
           Gear store closes { gearSaleEnd }.
           <br />
-          Shirts will be ordered on Monday, April 11 and assuming no supply chain delays, should be shipped out or ready for pick-up the week of April 18.
+          Shirts will be ordered on Monday, April 17 and assuming no supply chain delays, should be shipped out or ready for pick-up the week of April 24.
```
Comment (richardgolding) [inline client/components/public/Gear.jsx:230]: Not for 2023, but maybe in future: should we make more of the gear dates be configuration settings, similar to eventYear?

File: client/components/public/Gear.jsx (hunk 1 above)
Comment (RvanB) [reply client/components/public/Gear.jsx]: Good catch! Updated just now.

File: client/components/public/Gear.jsx (hunk 2 above)
Comment (RvanB) [reply client/components/public/Gear.jsx:230]: I agree, something like `gearOrderDate` and `gearShipDate`.

**Review by richardgolding — APPROVED** [review]
Looks good to me

**Review by cabeese — APPROVED** [review]
Awesome! thank you for doing this :)

**Pull request merged**